.vib/rails/goss/rails.yaml

```diff
@@ -0,0 +1,21 @@
+command:
+  # A new Rails project can be created
+  check-project:
+    timeout: 120000
+    exec: "rails new /app && cd /app && timeout 10s rails s -p 4000 || true"
+    exit-status: 0
+    stdout:
+      - Listening
+  check-app-version:
+  # rails --version returns other version format than APP_VERSION
```

Comment:
It'll be great if you could expand this comment a bit and give an overview of the syntax used by the asset here.